Modules/Filtering/BiasCorrection/include/itkN4BiasFieldCorrectionImageFilter.h

```diff
@@ -374,6 +374,12 @@ class N4BiasFieldCorrectionImageFilter : public ImageToImageFilter<TInputImage,
    */
   itkGetConstMacro(CurrentLevel, unsigned int);
 
+  /**
+   * Reconstruct bias field given the control point lattice.
+   */
+  RealImagePointer
+  ReconstructBiasField(BiasFieldControlPointLatticeType *);
+
```

Comment:
Was this intentionally made `public`?

Reply:
Yes. If this remained private, if a user wants to apply a bias field to a different image (e.g. a full resolution image), they first have to reconstruct the bias field. Which requires duplicating this method. It is better to just set a new input image to N4 filter, and call this method. Perhaps I should update the test to show how it reduces code duplication.